### DISCUSSION TITLE
Vmware 重启之后1panel无法查看用户名及密码

### DISCUSSION THREAD

**Discussion by fpd3210** [1Panel-dev/1Panel#5798] [Q&A]
![image](https://github.com/user-attachments/assets/8e9a17c1-d025-4299-8d2b-a7416d74d384)
Vmware 重启之后1panel无法查看用户名及密码，如上图

**Comment by wanghe-fit2cloud** [comment]
首次登录 1Panel 系统后，将不再支持明文查看密码。不过，你可以通过提示的命令手动重置密码。
